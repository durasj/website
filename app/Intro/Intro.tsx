import React from 'react';

export default class Intro extends React.Component<{}, {}>
{
    constructor(props: {}) {
        super(props);
    }

    render(): React.ReactElement<{}> {
        return (
            <div id="intro">
                <div>
                    Hi, I am <h1>Jakub, Web developer</h1>
                </div>
            </div>
        );
    }
}
