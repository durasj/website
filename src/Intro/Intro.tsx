import React, { PureComponent, ReactElement } from 'react';

import DocumentTitle from '../DocumentTitle';

export default class Intro extends PureComponent<{}, {}> {
    constructor(props: {}) {
        super(props);
    }

    public render(): ReactElement<{}> {
        return (
            <div className="intro">
                <DocumentTitle title="Jakub Duras - Software Developer" />
                <img src="icons/favicon.svg" alt="Lambda Jakub Duras logo" width="160px" height="160px" />
                <div className="introduction">
                    Hi, I am <h1>Jakub, Software developer</h1>
                </div>
            </div>
        );
    }
}
