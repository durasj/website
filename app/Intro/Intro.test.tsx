import React from 'react';
import renderer from 'react-test-renderer';

import Intro from '../Intro';

it('Renders Intro properly', () => {
    const component = renderer.create(
        <Intro />,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
