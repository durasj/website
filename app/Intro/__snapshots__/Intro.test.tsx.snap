// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Intro properly 1`] = `
<div
  className="intro"
>
  <img
    alt="Lambda Jakub Duras logo"
    height="160px"
    src="icons/favicon.svg"
    width="160px"
  />
  <div
    className="introduction"
  >
    Hi, I am 
    <h1>
      Jakub, Web developer
    </h1>
  </div>
</div>
`;
